tests/robot-tests/tests/admin/bau/prerelease_and_amend.robot

```diff
@@ -0,0 +1,302 @@
+Resource            ../../libs/tables-common.robot
+
+Suite Setup         user signs in as bau1
+Suite Teardown      user closes the browser
+Test Setup          fail test fast if required
+
+Force Tags          Admin    Local    Dev    AltersData
+
+
+*** Variables ***
+${TOPIC_NAME}=          %{TEST_TOPIC_NAME}
+${PUBLICATION_NAME}=    UI tests - prerelease and amend %{RUN_IDENTIFIER}
+${DATABLOCK_NAME}=      UI test table
+${RELEASE_NAME}=        Calendar Year 2000
+
+
+*** Test Cases ***
+Create test publication and release via API
+    ${PUBLICATION_ID}=    user creates test publication via api    ${PUBLICATION_NAME}
+    user creates test release via api    ${PUBLICATION_ID}    CY    2000
+
+Upload subject
+    user navigates to draft release page from dashboard    ${PUBLICATION_NAME}
+    ...    Calendar Year 2000
+
+    user uploads subject    UI test subject    upload-file-test.csv    upload-file-test.meta.csv
+
+Add metadata guidance
+    user clicks link    Data guidance
+    user waits until h2 is visible    Public data guidance
+
+    user waits until page contains element    id:dataGuidance-dataFiles
+
+    user checks summary list contains    Filename    upload-file-test.csv
+    user checks summary list contains    Geographic levels
+    ...    Local Authority; Local Authority District; Local Enterprise Partnership; Opportunity Area; Parliamentary Constituency; RSC Region; Regional; Ward
+    user checks summary list contains    Time period    2005 to 2020
+
+    user opens details dropdown    Variable names and descriptions
+
+    user checks table column heading contains    1    1    Variable name    css:table[data-testid="Variables"]
+    user checks table column heading contains    1    2    Variable description    css:table[data-testid="Variables"]
+
+    user checks table cell contains    1    1    admission_numbers    id:dataGuidance-dataFiles
+    user checks table cell contains    1    2    Admission Numbers    id:dataGuidance-dataFiles
+
+    user enters text into element    id:dataGuidanceForm-content    Test metadata guidance content
+    user enters text into element    id:dataGuidanceForm-subjects-0-content    Test file guidance content
+
+    user clicks button    Save guidance
+    user waits until page contains button    Edit guidance
+
+Add public prerelease access list
+    user clicks link    Pre-release access
+    user waits until h2 is visible    Manage pre-release user access
+    user creates public prerelease access list    Initial test public access list
+
+Go to "Sign off" page
+    user clicks link    Sign off
+    user waits until h2 is visible    Sign off
+    user waits until page contains button    Edit release status
+
+Check that there is no Pre-release warning text on the sign off page
+    user clicks link    Sign off
+    user waits until h2 is visible    Sign off
+    user waits until page contains button    Edit release status
```

Comment:
I think that everywhere we use `user clicks button`, we don't actually need to put a `user waits until page contains button` keyword before it.

This is because under the hood, the `user clicks button` keyword itself also uses `user waits until parent contains element` to make sure the button is present on the page before it is clicked.

I appreciate that you probably found this code snippet from another test that was already doing this, but maybe we can remove the `user waits until page contains button` keywords from this particular test script if it's immediately followed by `user clicks button`?